gpii/node_modules/lifecycleActions/src/LifecycleActions.js

```diff
@@ -43,3 +46,33 @@ fluid.defaults("gpii.launch.spawn", {
 
 gpii.launch.spawn = child_process.spawn;
 
+fluid.defaults("gpii.lifecycleActions.pathExists", {
+    gradeNames: "fluid.function",
+    argumentMap: {
+        path: 0
+    }
+});
+
+gpii.lifecycleActions.pathExists = function (path) {
+    if (!path) {
+        fluid.fail("No path was given as option to gpii.lifecycleActions.pathExists");
+    }
+    return fs.existsSync(path);
+};
+
+fluid.defaults("gpii.lifecycleActions.createFile", {
```

Comment:
Any tests for these!
Wouldn't we prefer to use one of the file-based settingsHandlers for this purpose? We should ensure that there is some syntax available that makes this possible.